.github/workflows/mql-mimic-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: MQL Mimic Tests
+
+on:
+  push:
+    branches: [ "**" ]
+
+concurrency:
+  # For pull_request_target workflows we want to use head_ref -- the branch triggering the workflow. Otherwise,
+  # use ref, which is the branch for a push event.
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trigger-via-zapier:
+    name: Trigger Test Run
+    runs-on: ubuntu-20.04
+    permissions:
+      checks: write
+
+    steps:
+
+      - name: "Trigger MQL Mimic Tests"
+        env:
+          trigger_url: '${{ secrets.MQL_MOCK_TRIGGER }}'
+          branch: '${{ github.ref_name }}'
+          repo: '${{ github.repository }}'
+          token: '${{ secrets.GITHUB_TOKEN }}'
+          sha: '${{ github.sha }}'
+        run: |
+          echo '{"branch":"'$branch'","repo":"'$repo'","token":"'$token'","sha":"'$sha'"}'
+          
+          curl -X POST $trigger_url  \
+            -H 'Content-Type: application/json' \
+            -d '{"branch":"'$branch'","repo":"'$repo'","token":"'$token'","sha":"'$sha'"}'
+
+      - name: Wait for check to be completed
```

Comment:
```suggestion
      - name: Wait for check to be completed
      # Wait for results so that the token remains valid while the test suite is executing and posting a check here.
```